src/plugins/data_source/server/saved_objects/data_source_saved_objects_client_wrapper.ts

```diff
@@ -116,100 +114,139 @@ export class CredentialSavedObjectsClientWrapper {
 
     return {
       ...wrapperOptions.client,
-      create: createWithCredentialMaterialsEncryption,
-      bulkCreate: bulkCreateWithCredentialMaterialsEncryption,
+      create: createWithCredentialsEncryption,
+      bulkCreate: bulkCreateWithCredentialsEncryption,
       checkConflicts: wrapperOptions.client.checkConflicts,
       delete: wrapperOptions.client.delete,
       find: wrapperOptions.client.find,
       bulkGet: wrapperOptions.client.bulkGet,
       get: wrapperOptions.client.get,
```

Comment:
Do we need to overwrite these to not to return credentials?

Reply:
not here, https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2035